src/listener/default-listener.js

```diff
@@ -0,0 +1,41 @@
+/*
+(c) Copyright 2016-2017 Hewlett Packard Enterprise Development LP
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+import Listener from "./base-listener";
+
+export default class DefaultListener extends Listener {
+  constructor(robot, transform) {
+    super(robot, undefined, transform);
+
+    this.catchAll(::this.Fallback);
+
+    }
+
+  Fallback(msg) {
+    //if msg.done is true then the bot was not referenced in the text
+    if (!msg.done) {
+      return this.transform.text(msg, "I'm sorry, I didn't quite get that. Try typing 'help' for a list of my capabilities.");

```

Comment:
Instead of:  "I didn't quite get that"  you might say:  "I didn't understand that."